arapp.json

```diff
@@ -22,11 +22,23 @@
       "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
       "network": "rinkeby"
     },
+    "staging": {
+      "registry": "0x98df287b6c145399aaa709692c8d308357bc085d",
+      "appName": "placeholder-app-name-staging.open.aragonpm.eth",
+      "wsRPC": "wss://rinkeby.eth.aragon.network/ws",
+      "network": "rinkeby"
+    },
     "mainnet": {
       "registry": "0x314159265dd8dbb310642f98f50c066173c1259b",
       "appName": "placeholder-app-name.open.aragonpm.eth",
       "wsRPC": "wss://mainnet.eth.aragon.network/ws",
       "network": "mainnet"
+    },
+    "ropsten": {
+      "registry": "0x6afe2cacee211ea9179992f89dc61ff25c61e923",
```

Comment:
As above, https://github.com/aragon/deployments/blob/master/environments/ropsten/readme.md

Reply:
This one seems right 🤔